test/geneexpr_test.jl

```diff
@@ -31,15 +31,21 @@ function runSSAs(jump_prob; use_stepper = true)
     mean(Psamp)
 end
 
-function runSSAs_ode(jump_prob)
-    Psamp = zeros(Int, Nsims)
+function runSSAs_ode(oprob, vrjs, vr_agg)
+    Psamp = zeros(Float64, Nsims)
     for i in 1:Nsims
-        sol = solve(jump_prob, Tsit5(); saveat = jump_prob.prob.tspan[2])
-        Psamp[i] = sol[3, end]
+        vrjprob = JumpProblem(oprob, vrjs; vr_aggregator = vr_agg, save_positions = (false, false), rng)
```

Comment:
This is changing the test. The whole point is it should work if you just call solve repeatedly. That you are needing to recreate the JumpProblem is indicative that it isn't re-initializing correctly.